packages/scss/src/components/_table.scss

```diff
@@ -187,6 +187,26 @@
 			}
 		}
 	}
+
+	&.mod-stickyHeader {
+		.table-head-row-cell {
+			background-color: _color("white");
+			position: sticky;
+			z-index: 1;
+		}
+	}
+
+	&.mod-top0 {
+		.table-head-row-cell {
+			top: 0;
```

Comment:
Do we know which one is more used (top0 or top 50px) ?
We must add the top: xx in the mod-stickyheader because without it, the mod doesn't work. 
It it would make only one modifier